microcosm_elasticsearch/store.py

```diff
@@ -56,12 +74,12 @@ def flushing(self):
 
         """
         yield
-        self.index.flush()
+        self.get_index(**kwargs).flush()
         # NB. as of ES7 a flush does not have the side-effect of refresh.
         # Given that our use of explicit flush in code typically is done for refreshing
         # available documents to be visible to the search engine, we also invoke refresh below.
         # See: https://qbox.io/blog/refresh-flush-operations-elasticsearch-guide
-        self.index.refresh()
+        self.get_index(**kwargs).refresh()
```

Comment:
nit: it can be a single call to `get_index`, then `index.flush()` and `index.refresh()`